lib/arjdbc/postgresql/adapter.rb

```diff
@@ -218,15 +218,23 @@ def standard_conforming_strings?
       @standard_conforming_strings == true # return false if :unsupported
     end
 
-    # Does PostgreSQL support migrations?
-    def supports_migrations?
-      true
-    end
+    def supports_ddl_transactions?; true end
 
-    # Does PostgreSQL support finding primary key on non-Active Record tables?
-    def supports_primary_key?
-      true
-    end
+    def supports_explain?; true end
```

Comment:
This was moved here from the explain_support.rb file. The others were just moved around so that I could group them into once place if they are just returning `true`.